### DISCUSSION TITLE
Question about Exchange Spec.Name vs. ObjectMeta.Name

### DISCUSSION THREAD

**Discussion by vaikas** [rabbitmq/rabbitmq-server#2931] [Questions] [answered]
Is this field:
https://github.com/rabbitmq/messaging-topology-operator/blob/main/api/v1alpha2/exchange_types.go#L21

Different from Exchange.ObjectMeta.Name?

If not, just curious if it's needed as it seems redundant. If it is, how does it handle collisions if two CRs create an exchange with different k8s name but use the same Spec.Name field?

Thanks!

**Comment by michaelklishin** [comment]
@ChunyiLyu @MirahImage this was started here because messaging topology Operator does not have Discussions enabled

**Reply by MirahImage** [reply]
I've enabled discussions on the messaging topology operator now

**Comment by michaelklishin** [comment, marked as answer]
@vaikas may I ask you to move this to the message topology Operator repo? https://github.com/rabbitmq/messaging-topology-operator/discussions

**Reply by vaikas** [reply]
Thanks @MirahImage and @michaelklishin moved here:
https://github.com/rabbitmq/messaging-topology-operator/discussions/100
